lib/commonBindings.js

```diff
@@ -115,7 +115,7 @@ function manageConfigurationRequest(apiKey, deviceId, device, objMessage) {
  * @param {Number} index        Index of the group in the array.
  * @return {Array}             Updated array of functions.
  */
-function processMeasureGroup(device, apikey, previous, current, index) {
+function processMeasureGroup(device, apikey, previous, current) {
```

Comment:
JavaDoc entry for index (L115) should be removed also.

Reply:
fixed in 662a45864574e93727ea7e72501236688c2d9d33